### ISSUE TITLE
Changing bg and text color

### ISSUE THREAD

**Issue by sebambsch** [dixonary/fingerterm-reMarkable#6]
I tried to change the background and text color of the terminal. In my opinion black writing on white screen is more compatible with the eInk technology used (as far as I know white is the default - any else has Infos on that?). I changed the bgcolor argument in the Main.qml file to white (#ffffff) and when opening fingerterm the background is indeed white. However as soon as I start typing anything and the page get's updated it reverts to black with white text. Is there anything in the update routine that overwrites the bgcolor argument?
p.s. otherwise I love fingerterm :) Very happy to be able to have a terminal on my rM

**Comment by dixonary** [comment]
You're right, the default mode [for fingerterm] is white on black. The reason I switched it to black on white is that (experimentally) it reduced the amount of complete redraws that the device did. 

The firmware is designed to flash the screen black occasionally to clear any dark ghosting. When drawing black, this flash is much less obvious and I think this makes it more comfortable to use.

I made this change only by editing the background color in the source, so I'm not sure why you are having trouble reversing this change I'm afraid :(
